packages/mui-base/src/SliderUnstyled/SliderValueLabelUnstyled.d.ts

```diff
@@ -11,11 +12,15 @@ export interface ValueLabelUnstyledProps {
   components?: {
     Root?: React.ElementType;
   };
+  /**
+   * If `true`, the value label is visible.
+   */
+  open: boolean;
   /**
    * The value of the slider.
    * For ranged sliders, provide an array with two values.
    */
-  value?: number | number[];
+  value: number;
```

Comment:
This type was false, it might actually be 
```suggestion
  value: number | string;
```

but it's another topic, ` | number[]` was plain wrong.

Reply:
Correct, there could be only one value for the `ValueLabel`. Keeping it just `number` for now sounds good.